tests/test_alchemy.py

```diff
@@ -261,3 +261,34 @@ def test_build_does_not_raises_exception_when_no_session_was_set(self):
         inst1 = NoSessionFactory.build()
         self.assertEqual(inst0.id, 0)
         self.assertEqual(inst1.id, 1)
+
+
+class NameConflictTests(unittest.TestCase):
+    """Regression test for `TypeError: _save() got multiple values for argument 'session'`
+
+    See #775.
+    """
+    def test_no_name_conflict_on_save(self):
+        class SpecialFieldWithSaveFactory(SQLAlchemyModelFactory):
+            class Meta:
+                model = models.SpecialFieldModel
+                sqlalchemy_session = models.session
+
+            id = factory.Sequence(lambda n: n)
+            session = ''
+
+        saved_child = SpecialFieldWithSaveFactory()
+        self.assertIsNotNone(saved_child.session)
```

Comment:
Might as well be clear about the expected result:
```suggestion
        self.assertEqual(saved_child.session, "")
```